### ISSUE TITLE
Loss function for NER task

### ISSUE THREAD

**Issue by udion** [ZhixiuYe/HSCRF-pytorch#6]
Can you please point to some reference which you used to implement the loss function for NER task?

I think your paper only discusses loss function for the word level segmenting.

Thanks

**Comment by ZhixiuYe** [comment]
In my paper, I discusse the loss function for the segment level loss by HSCRF, and omit the CRF loss.

**Comment by udion** [comment]
Almost always people use negative log likelihood for CRF layer, right?

**Comment by ZhixiuYe** [comment]
Yes.
